pandas/core/frame.py

```diff
@@ -247,11 +247,13 @@ class DataFrame(NDFrame):
 
     Examples
     --------
+    >>> import numpy as np
+    >>> import pandas as pd
```

Comment:
Those two are not needed (we assume those, to not have to repeat those in each docstring. Also the tutorial docs everywhere assume those two imports)